README.md

```diff
@@ -31,7 +31,10 @@ Good luck.
 - Python
   - [Python Documentation](https://www.python.org/doc/)
   - [Flask](https://flask.palletsprojects.com/en/1.1.x/)
-    - Pycharm
+ - Pycharm
+      - [Configure the desired shell of terminal emulator](https://www.jetbrains.com/help/pycharm/terminal-emulator.html#configure-the-terminal-emulator)
+      - [Pycharm keyboard shortcuts](https://www.jetbrains.com/help/pycharm/mastering-keyboard-shortcuts.html) 
+
```

Comment:
Hi Chin Li!

Line 35 is a duplicate of line 38. Please remove line 35.

Also, could you move Pycharm back under Python? As this would be the basic introduction of Devops, it would make more sense for it to be placed under Python as an IDE subject.

An example:
```
Python
      - Python Documentation
      - Flask
      - IDE for programming (Integrated Development Environment)
            - Pycharm
                   -  xxxx
                   -  xxxx
                 
            -  VSC (Visual Source Code)
                   -  xxxx
```

Do let me know if you have any opinions on this! :)